app/src/main/java/com/example/androidtraining/networking/TokenProvider.kt

```diff
@@ -0,0 +1,25 @@
+package com.example.androidtraining.networking
+
+class TokenProvider(
+    private var jwtToken: String? = null,
+    private var isLoggedIn: Boolean = false
+) {
+
+    fun setJwtToken(token: String) {
+        jwtToken = token
```

Comment:
What if we want to clear the token? After all, the local variable is nullable. Additionally, the separate boolean is also not really needed -- either the user has a token, or s/he doesn't.

Reply:
I have created fun clearToken() which I intend to use after logout. Or I can use setJwtToken with null value.
For additional bool I agree - will remove it.